src/ucp/wireup/address.h

```diff
@@ -40,7 +40,7 @@ enum {
 struct ucp_address_iface_attr {
     uint64_t                    cap_flags;    /* Interface capability flags */
     double                      overhead;     /* Interface performance - overhead */
-    double                      bandwidth;    /* Interface performance - bandwidth */
+    uct_tl_bandwidth_t          bandwidth;    /* Interface performance - bandwidth */
```

Comment:
no need to change, as you have 1 value now

Reply:
there used nested value to prevent direct access to this value (without interpretation by +/-)